### ISSUE TITLE
Enable a11y rule in eslint

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/navigation#178]
- Enable a11y rules in eslint config: https://github.com/gravity-ui/eslint-config?tab=readme-ov-file#a11y and disable if the issue came from rule cannot be fixed
- Provide an issue for each disabled rule with instructions on how to fix it
